### DISCUSSION TITLE
[Projects beta] filter issues by status

### DISCUSSION THREAD

**Discussion by CFGrote** [community/community#11707] [Projects and Issues]
When searching for an issue in a repo, I'd like to see only open issues. E.g.
```
repo:foo is:open
```

**Comment by elyobo** [comment]
Agreed, the lack of filters is very confusing. I'm sure that some of these _used_ to work as well, I went through and manually added a bunch of issues and was able to select only issues (don't want PRs in our project) and only open (don't care about already closed) but there doesn't seem to be a way to do this any more.

The  old projects [supported a lot of filtering options](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/tracking-work-with-project-boards/filtering-cards-on-a-project-board), that seems like a good starting point.

Lack of filtering [seems to be a common complaint here](https://github.com/github/feedback/discussions/categories/issues-feedback?discussions_q=category%3A%22Issues+Feedback%22+filter) but isn't getting much in the way of responses.
